src/olympia/addons/tests/test_commands.py

```diff
@@ -383,9 +386,17 @@ def test_migrate_preliminary(
     addon.reload()
     assert addon.status == end_addon_status
     assert addon.is_experimental == end_experimental
+    # Check v1's status matches
     v1 = Version.objects.filter(addon=addon, version='1')[0]
     assert v1.all_files[0].status == end_file1_status
-    assert addon.latest_version.all_files[0].status == end_file2_status
+    # Now check original_status worked too
+    v2 = Version.objects.filter(addon=addon, version='2')[0]
```

Comment:
Or just `Version.objects.get(addon=addon, version='2')` (also applies for `v1` above)